### DISCUSSION TITLE
Color mode transition

### DISCUSSION THREAD

**Discussion by runoncedev** [chakra-ui/chakra-ui#4501] [Q&A] [answered]
First of all, thank you for such an amazing work. I love Chakra UI!

Now, I'm trying to sync up a CSS background color transition with the [Color Mode](https://chakra-ui.com/docs/features/color-mode) transition.

Let's say I have this:

```javascript
export const Header = () => {
  const background = useColorModeValue("red", "blue");

  return (
    <Box transition="background" background={background}>
      Hello there!
    </Box>
  );
};
```

And I want to match the duration and easing of the background transition with the Color Mode transition.

I checked on the theming section of the docs but couldn't find any relevant information.

Any suggestions?

**Comment by primos63** [comment, marked as answer]
You may have already found the answer to your question, but in case you did not:
The global styles for the default theme, `packages/theme/src/styles.ts` have the settings I think you are looking for.
```javascript
    body: {
      fontFamily: "body",
      color: mode("gray.800", "whiteAlpha.900")(props),
      bg: mode("white", "gray.800")(props),
      transitionProperty: "background-color",
      transitionDuration: "normal",
      lineHeight: "base",
    }
 ```

**Reply by runoncedev** [reply]
I didn't! Thank you, I will try this out.
